test/Microsoft.AspNetCore.Antiforgery.FunctionalTests/AntiforgerySampleTest.cs

```diff
@@ -26,11 +27,19 @@ public AntiforgerySampleTests(AntiForgerySampleTestFixture fixture)
             var response = await Client.GetAsync("http://localhost/Index.html");
 
             // Assert
-            var cookie = RetrieveAntiforgeryCookie(response);
-            Assert.NotNull(cookie.Value);
-
-            var token = RetrieveAntiforgeryToken(response);
-            Assert.NotNull(token.Value);
+            var setCookieHeaderValue = RetrieveAntiforgeryCookie(response);
+            Assert.NotNull(setCookieHeaderValue);
+            Assert.False(string.IsNullOrEmpty(setCookieHeaderValue.Value));
```

Comment:
I do not see a specific assertion for this...we can do in 2 steps..check for notnull and assert.notempty...